doc/release/release_dev.rst

```diff
@@ -26,7 +26,10 @@ Improvements
 API Changes
 -----------
 
-
+- All references to EN-GB spelling for the word ``neighbour`` and others—e.g.,
+  ``neigbourhood``, ``neighboring``, were changed to their EN-US spelling,
+  ``neighbor``. With that, ``skimage.measure.perimeter` parameter ``neighbourhood``
+  was deprecated in favor of ``neighborhood`` in 1.0.
```

Comment:
This line indicates the deprecation is introduced in 1.0 which is inconsistent with the decorator which states that the argument is removed in 1.0.